src/Resources/contao/config/config.php

```diff
@@ -33,66 +34,74 @@
     $GLOBALS['FE_MOD'],
     2,
     [
-        'wem-job-offers' => [
-            'jobslist' => 'WEM\JobOffersBundle\Module\ModuleJobOffersList',
-            'jobsalert' => 'WEM\JobOffersBundle\Module\ModuleJobOffersAlert',
+        'wem-offers' => [
+            'offerslist' => 'WEM\OffersBundle\Module\ModuleOffersList',
+            'offersalert' => 'WEM\OffersBundle\Module\ModuleOffersAlert',
         ],
     ]
 );
 
-// Load icon in Contao 4.2 backend
-if ('BE' === TL_MODE) {
-    if (version_compare(VERSION, '4.4', '<')) {
-        $GLOBALS['TL_CSS'][] = 'bundles/joboffers/backend/backend.css';
-    } else {
-        $GLOBALS['TL_CSS'][] = 'bundles/joboffers/backend/backend_svg.css';
-    }
-}
-
 // Hooks
-$GLOBALS['TL_HOOKS']['loadDataContainer'][] = [WEM\JobOffersBundle\Hooks\LoadDataContainerHook::class, 'addAttributesToJobDca'];
-$GLOBALS['TL_HOOKS']['storeFormData'][] = [WEM\JobOffersBundle\Hooks\StoreFormDataHook::class, 'storeFormData'];
-$GLOBALS['TL_HOOKS']['processFormData'][] = [WEM\JobOffersBundle\Hooks\ProcessFormDataHook::class, '__invoke'];
+$GLOBALS['TL_HOOKS']['loadDataContainer'][] = [WEM\OffersBundle\Hooks\LoadDataContainerHook::class, 'addAttributesToJobDca'];
```

Comment:
As we're  in a code refactoring, shouldn't we use the new syntax `offers.listener.xxxx` ?

Reply:
Fixed in 67de83cbdbf7a6bc09ec197355649082da765d9d